app/assets/purs/src/EntryPoints/Comments.purs

```diff
@@ -66,11 +74,35 @@ def props =
         forkVoid (createComment props.postId state.commenter state.newComment)
         pure state 
           {
-            comments = Array.snoc state.comments { commenter: state.commenter, comment: state.newComment }
+            comments = Array.snoc state.comments { commenter: state.commenter, comment: state.newComment, upvotes: 0, downvotes: 0, id: 1 + (Array.length state.comments) }
```

Comment:
This looks like one of those cases where we care about the API call response. I tried to do it, but couldn't find any good examples in the main codebase. Here's my attempt in case you're interested: https://github.com/daviddzhang/launchpad/commit/ebbe17a2cb194b46870365ec37b7208b0ea3d167